funnel/templates/account_merge.html.jinja2

```diff
@@ -4,21 +4,21 @@
 
 {% macro accountinfo(user) %}
   <ul class="mui-list--aligned mui--text-subhead">
-    <li><strong>{% trans name=user.fullname %}Name:</strong> {{ name }}{% endtrans %}</li>
-    <li><strong>{% trans username=user.username or '' %}Username:</strong> {{ username }}{% endtrans %}</li>
+    <li><span class="mui--text-bold">{% trans name=user.fullname %}Name:{% endtrans %}</span> {{ name }}</li>
+    <li><span class="mui--text-bold">{% trans username=user.username or '' %}Username:{% endtrans %}</span> {{ username }}</li>
```

Comment:
The parameter to `trans` is no longer required. Both variables need to be expanded as they are no longer defined from the `trans` tag.